dashboard/src/lib/string.ts

```diff
@@ -8,3 +8,25 @@ export const truncateBigText = (
   text && text.length > maxTextLength
     ? text.slice(0, maxTextLength) + '...'
     : valueOrEmpty(text);
+
+export const truncateUrl = (
+  url: string | undefined,
+  domainLength = 50,
+  endPathLength = 20,
+): string => {
+  if ('parse' in URL && url) {
```

Comment:
If 'parse' not in URL (meaning the browser does not support this) but url is not undefined, the function should call truncateBigText so that the URL is not shown unprocessed. Other solution would be to refactor this function to always use native/supported JS functions like split('/')